spec/lib/webfinger_spec.rb

```diff
@@ -4,28 +4,28 @@
 
 require 'spec_helper'
 
-describe Webfinger do
+describe Diaspora::Webfinger do
   let(:host_meta_xrd) { File.open(Rails.root.join('spec', 'fixtures', 'host-meta.fixture.html')).read }
   let(:webfinger_xrd) { File.open(Rails.root.join('spec', 'fixtures', 'webfinger.fixture.html')).read }
   let(:hcard_xml) { File.open(Rails.root.join('spec', 'fixtures', 'hcard.fixture.html')).read }
   let(:account){'foo@bar.com'}
   let(:account_in_fixtures){"alice@localhost:9887"}
-  let(:finger){Webfinger.new(account)}
+  let(:finger){Diaspora::Webfinger.new(account)}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.